File: 프로그래머스/0/120811. 중앙값 구하기/중앙값 구하기.java
import java.util.Arrays;

class Solution {
    public int solution(int[] array) {
        Arrays.sort(array);
        
        int answer = 0;
        
        if (array.length == 3) {
            answer = array[array.length%2];
        } else {
            answer = array[array.length/2];
        }
        return answer;
    }
}

File: 프로그래머스/0/120811. 중앙값 구하기/중앙값 구하기.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>


int solution(int array[], size_t array_len) {
    int answer = 0;
    int save = 0; 
    
    for (int i = 0; i < array_len; i++) {
        for (int j = i; j < array_len; j++) {
            if (array[i] > array[j]) {
                save = array[i];
                array[i] = array[j];
                array[j] = save;
            }
        }
    }
    
    return array[array_len/2];
}